portfolio/src/main/webapp/index.html

```diff
@@ -5,8 +5,9 @@
     <title>Yuntian Wu</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    
   </head>
-  <body>
+  <body onload="showString()">
```

Comment:
this is a typo, already changed to 'body onload="printString()"'

Reply:
Good job finding the bug!